### ISSUE TITLE
Content Publishing - Pattern (06 Content Group - Horizontal)

### ISSUE THREAD

**Issue by JennySanchez** [carbon-design-system/ibm-dotcom-library-website#505]
#### User Story

> As an IBM.com Library pattern adopter, 

> I need a way to learn more about the (Name of the Pattern here) pattern 

> so that I can utilize the pattern to my day to day work.

#### Additional information

- Take a look at the [storybook](https://ibmdotcom-react.mybluemix.net/?path=/story/overview-getting-started--page) - so you know how the pattern works.   You will also use Storybook as a reference to accurately include the images in the Sketch file.
- Download this [Sketch template](https://ibm.box.com/s/dgmmw2kpvtt3hpw8bazprawpfxfvsac3) to your machine to make the updates. Name the Sketch file with the following convention: Name of the pattern + version. ex: `Feature card block v1.0`
- Use the pattern description included in the [Patterns Overview Box Note](https://ibm.box.com/s/wtuzuvk7a0nxbtimo341a8te1sb6w6h0) - for the 'Intro' section of the newly downloaded Sketch file
- Navigate to the visual spec file within this [Patterns Box folder](https://ibm.box.com/s/dfmbd79ek0hmexfz1avg595njmzb2bap) - The image that works successfully for most patterns is the 1312px image.  Adjustments may be needed to have this image accurately reflect the Storybook designs.
- Once complete, save the revised Sketch file in the Layout Pattern folder / sub-folder [here](https://ibm.box.com/s/jao6qplwburqkn1vjrrrvxum1qn3dsoc).
- Export the individual images with a 2X setting, saving them into the Layout Pattern sub-folder that contains the Sketch file.  These exported images will be used when creating the Markdown for publishing
 - Use this Sketch file to create the Markdown version of the web page in [Github](https://github.com/carbon-design-system/ibm-dotcom-library-website/tree/master/src/pages/patterns).
    - Find the .mdx file for the Layout Pattern assigned
    - Copy the content from the Sketch file into the .mdx file
    - Once complete, create the Pull Request and review, using the Preview link, to be sure it matches the Sketch file design
    - When ready to deploy, include Jenny as a PR Reviewer and Approver.
    - Jenny will review and merge the PR

#### Acceptance criteria
- [x] Content crafting is done.
- [x] Asset image creation is done.
- [x] Page design is done.
- [x] Publishing with markdowns is done.
- [x] QA'd by Jenny
- [x] Web page preview is approved by Linda, Jeff and Wonil
- [x] Web page is published

**Comment by JennySanchez** [comment]
**Content group — horizontal**

👉 Work files can be found in the [Callout quote BOX folder](https://ibm.box.com/s/rmfpuc1rcsgtt2a84opx7g784eitam1p).

![Pattern - Content group — horizontal.jpg](https://images.zenhubusercontent.com/5da0db0a9f0a9500014f53bf/74ef2870-9dd5-4804-91fb-d395ffebdea0)

**Comment by JennySanchez** [comment]
@ljcarot @wonilsuhibm @jeffchew 

Content markdown completed and the preview link can be found in [PR #546](https://github.com/carbon-design-system/ibm-dotcom-library-website/pull/546

**Comment by wonilsuhibm** [comment]
@JennySanchez thank you.

A question: why is the logo grid pattern called logo grid — rich?

**Comment by wonilsuhibm** [comment]
@JennySanchez re the CTA section, I know we decided to not touch the images, but this is just too much. Will it be possible to reduce the amount of negative space on either side?

<img width="1183" alt="Screen Shot 2020-07-29 at 5 36 04 PM" src="https://user-images.githubusercontent.com/30183447/88856170-053d8580-d1c2-11ea-9fa0-a75623ea843b.png">

**Comment by JennySanchez** [comment]
@wonilsuhibm The "Logo grid — rich" is what listed in the [Aritable inventory](https://airtable.com/tblsnDDtb57UVxHpH/viwL7XPwcIgnb8Y4T?blocks=hide) as an updated name to the "logo grid.

![Screen Shot 2020-07-29 at 5.12.31 PM.png](https://images.zenhubusercontent.com/5da0db0a9f0a9500014f53bf/e91fc6de-3dec-467f-b503-14be638535ae)

**Comment by JennySanchez** [comment]
@wonilsuhibm The comments above is for "Logo Grid". I am moving the conversation to the correct Git issue.

**Comment by wonilsuhibm** [comment]
@JennySanchez The screenshot has too much space on either side. Any chance we can crop it to show exactly the 16 columns?

**Comment by JennySanchez** [comment]
Let me add it to #558 and work on it.

**Comment by wonilsuhibm** [comment]
other than this is good to go.

**Comment by JennySanchez** [comment]
@wonilsuhibm We did not go with the actual 16 column image treatment, bc we talked about some patterns that are vertically really long to show the whole pattern and it will take up a lot of vertical space on the website page. For the MVP, we will sell it down to fit the width of 928 and adjust the vertical space to make sure it fits nicely visually. I did update the image so it won't have as many white spaces on both sides. ![content-group-horizontal.jpg](https://images.zenhubusercontent.com/5da0db0a9f0a9500014f53bf/9e5096a4-822c-4437-bdec-c9a721801d31)
